packages/components/scripts/vendorism/custom/translations.vendorism.js

```diff
@@ -18,6 +18,10 @@ const translations = {
     de: 'Menu öffnen',
     en: 'Open menu',
   },
+  priorityMenuLabel: {
```

Comment:
The translation should not be connected to an explicit component.

Just call the key "menu".

Reply:
Fixed